crates/diesel_models/src/query/merchant_connector_account.rs

```diff
@@ -227,4 +227,37 @@ impl MerchantConnectorAccount {
             .await
         }
     }
+
+    pub async fn find_by_profile_id(
+        conn: &PgPooledConn,
+        profile_id: &common_utils::id_type::ProfileId,
+        get_disabled: bool,
```

Comment:
do we need this param for the list query?

Reply:
Currently it's not used, but in other flows like payouts we would be needing this

Reply:
but this is just a list endpoint right? this is agnostic of whether payment or payout

Reply:
Yes, that flag is useful in few flows where we have to get connectors excluding disabled ones